biojava-structure/pom.xml

```diff
@@ -62,48 +62,41 @@
 
 		<dependency>
 			<groupId>org.jgrapht</groupId>
-  			<artifactId>jgrapht-core</artifactId>
-  			<version>1.5.1</version>
+			<artifactId>jgrapht-core</artifactId>
+			<!-- Pin! Java 8 requires 1.3.0 -->
+			<version>1.3.0</version>
```

Comment:
How about 1.4.0? It seems that would be the last java-8 compatible. That's what their changelog says: https://github.com/jgrapht/jgrapht/blob/master/HISTORY.md. I've double checked by downloading the jar and looking at a class file.

Reply:
Cood catch. I was looking at `Build-Jdk: 11.0.6` in the manifest, when of course I should have been checking `Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.8))"`. I will update to 1.4.0